docs/content/en/docs-dev/user-guide/managing-controlplane/configuration-reference.md

```diff
@@ -147,6 +147,7 @@ Must be one of the following objects:
 |-|-|-|-|
 | name | string | The unique name of the configuration. | Yes |
 | provider | string | The SSO service provider. Can be one of the following values<br>`GITHUB`, `GOOGLE`... | Yes |
+| sessionTtl | int | The time to live of session for SSO login. Unit is `minute`. Default is 7 hours. | No |
```

Comment:
I think the current default TTL is 7 days
ref: https://github.com/pipe-cd/pipecd/blob/master/pkg/app/server/httpapi/auth_handler.go#L51

Reply:
And looks like the current default time is in units of hours, so could you recheck around that part?

Reply:
Thank you, I have changed "minutes" to "hours"